addons/project/models/project_task.py

```diff
@@ -1308,7 +1308,11 @@ def write(self, vals):
             for task in self:
                 project_link = project_link_per_task_id.get(task.id)
                 if project_link:
-                    body = _('Task Transferred from Project %s', project_link)
+                    body = _(
+                        'Task Transferred from Project %(source_project)s to %(destination_project)s',
+                        source_project=project_link,
+                        destination_project=self.project_id._get_html_link(title=self.project_id.display_name),
```

Comment:
If a task is updated from project A into a private task, is the notification send anyway ? :thinking:  Won't this line crash in this case ?
If no, then you can green the task.

Reply:
Hello, I don't think so, since it only applies to when the task is moved to a project and the partners are subscribed to tasks being created in that project